tests/test_utils.py

```diff
@@ -100,4 +100,25 @@ def test_get_min_max_for_permitted_types(self):
         for value_type in permitted_numeric_types:
             (min_value, max_value) = get_min_max(value_type)
             self.assertEqual(min_value, expected_min_max_values[value_type][0])
-            self.assertEqual(max_value, expected_min_max_values[value_type][1])
\ No newline at end of file
+            self.assertEqual(max_value, expected_min_max_values[value_type][1])
+
+    def test_to_timestamp(self):
```

Comment:
I had originally tested a number of other dates to make sure those before and after the epoch worked for winter and summer time:

```
datetime(2020, 1, 4, 1, 2, 3)  # winter time
datetime(2020, 6, 4, 1, 2, 3)  # daylight savings time
datetime(2050, 1, 4, 1, 2, 3)  # winter time
datetime(2050, 6, 4, 1, 2, 3)  # daylight savings time
datetime(1969, 1, 4, 1, 2, 3)  # winter time
datetime(1969, 6, 4, 1, 2, 3)  # daylight savings time
```

I had tested dates before the epoch by subtracting a `timedelta` value from the epoch but you don't have to test things that way here. It's important that we verify winter and summer dates before the epoch, since the epoch but in the past, and in the future, all work correctly at least to within an hour if possible. Dates before 1910 we don't have to worry about, all dates between that and the epoch should all be equivalent as far as testing is concerned.